lib/r10k/content_synchronizer.rb

```diff
@@ -8,31 +8,40 @@ def self.serial_accept(modules, visitor, loader)
     end
 
     def self.serial_sync(modules)
+      updated_modules = []
       modules.each do |mod|
-        mod.sync
+        updated = mod.sync
+        updated_modules << mod.name if updated
       end
+      updated_modules
     end
 
+    # Returns a Queue of the names of modules actually updated
     def self.concurrent_accept(modules, visitor, loader, pool_size, logger)
       mods_queue = modules_visit_queue(modules, visitor, loader)
       sync_queue(mods_queue, pool_size, logger)
     end
 
+    # Returns a Queue of the names of modules actually updated
```

Comment:
fwiw, I think this will return either a Queue of module names, or nil, if there was an error during a deprecated `accept` call, or raise if there was an error during the newer `sync`s.

Reply:
Do you want me to update this? I didn't dig into these methods, but I can if you want.

Reply:
That's fine, I was more trying to tie together why we were checking for both `nil?` and `empty?` in the caller